addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/SubResourceIntegrityAttributeScanRuleUnitTest.java

```diff
@@ -91,11 +136,36 @@ void shouldRaiseAlertGivenIntegrityAttributeIsMissingForSupportedElement()
 
         // Then
         assertThat(alertsRaised.get(0).getPluginId(), equalTo(rule.getPluginId()));
+        assertThat(
+                alertsRaised.get(0).getOtherInfo(),
+                equalTo(
+                        "The following hash was calculated for the resource in question "
+                                + "3c9c30d9f665e74d515c842960d4a451c83a0125fd3de7392d7b37231af10c72ea58aedfcdf89a5765bf902af93ecf06"));
```

Comment:
That’s on me :wink:

Reply:
Yeah Ill change the unit tests. Also if I'm adding the "correct-hash" check functionality then I guess I'll have to add a new test. :)

Reply:
I believe you were going to revert this and add new tests.

Reply:
@thc202 we've been discussing this out-of-band. Were you thinking that something more/different was needed? This allows to ensure the other info comes through properly. I guess maybe it only needs to be added to one of the 3 test methods, but adding another test just for other info generation specifically seem like separation with little benefit? Unless I'm missing something...

Reply:
IMO it would be more clear in its own test, this test is about raising an alert if the element is missing not whether the value was properly generated, but I'm ok if kept in just one.